requirements/base.txt

```diff
@@ -1,10 +1,9 @@
-mock==2.0.0
-responses==0.10.15
-google-resumable-media==1.1.0
-googleapis-common-protos==1.6.0
-google-cloud-storage==1.32.0
-better-setuptools-git-version==1.0.5
-unittest-xml-reporting==3.0.2
-wheel==0.35.1
-setuptools==49.6.0
-jinja2<2.11.0,>=2.10.1
\ No newline at end of file
+google-cloud-storage>=1.32,<2
+google-auth>=1.20,<2
+
+better-setuptools-git-version>=1.0.5,<2
+unittest-xml-reporting>=3.0.2,<4
+jinja2>=2.10,<3
+
+wheel>=0.35
```

Comment:
same here, shouldn't we add the upper bound?

Reply:
AFAIK 'wheel' is also preinstalled even on fresh 'venv'. Also it is not directly used by 'bigflow', so it is better to allow 'setuptools' apply it's own restrictions.